qt_gui_cpp/include/qt_gui_cpp/composite_plugin_provider.h

```diff
@@ -33,52 +33,10 @@
 #ifndef qt_gui_cpp__CompositePluginProvider_H
 #define qt_gui_cpp__CompositePluginProvider_H
 
-#include "plugin_descriptor.h"
-#include "plugin_provider.h"
+// *INDENT-OFF* (prevent uncrustify from adding indention below)
+#warning Including header <qt_gui_cpp/composite_plugin_provider.h> is deprecated, \
+include <qt_gui_cpp/composite_plugin_provider.hpp> instead.
```

Comment:
If we have to do `INDENT-OFF`, we also need to remember to turn it back on.  This probably doesn't matter much in these header files, but we should just be consistent:

```suggestion
// *INDENT-OFF* (prevent uncrustify from adding indention below)
#warning Including header <qt_gui_cpp/composite_plugin_provider.h> is deprecated, \
include <qt_gui_cpp/composite_plugin_provider.hpp> instead.
// *INDENT-ON*
```

(same goes for all other `.h` header files below)